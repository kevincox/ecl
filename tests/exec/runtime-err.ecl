local ref = ..fart
local dict = { a = ref }
local add = dict.a + 2
func = add:1
