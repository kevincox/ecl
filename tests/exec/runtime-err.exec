cmd: load runtime-err.ecl -flines
status: 1
