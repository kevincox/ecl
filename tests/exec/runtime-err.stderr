At line 4.11 On left side of call
At line 4.8 Referenced by "add"
At line 3.20 On left side of addition
At line 2.20 Referenced by "ref"
At line 1.13 referenced by .."fart"
Can't lookup "fart" in Nil

Error occurred.
